eng/common/scripts/stress-testing/find-all-stress-packages.ps1

```diff
@@ -26,6 +33,17 @@ function FindStressPackages(
     foreach ($chartFile in $chartFiles) {
         $chart = ParseChart $chartFile
         if (matchesAnnotations $chart $filters) {
+            $matrixFilePath = (Join-Path $chartFile.Directory.FullName '/matrix.yaml')
```

Comment:
Maybe `scenario-matrix.yaml`?

Reply:
Also this should be customizable, I guess when you pull the matrix script pieces out.

Reply:
I'm not sure what you meant by customizable as we want to traverse the entire directory to find all stress packages

Reply:
the only thing that might be customizable is the matrix file name (or relative path) but not the entire directory, and this will force all nested packages to have the exact same filename/relative path

Reply:
Yes I mean the matrix filename. People might have multiple matrices defined and want to swap from one to the other in testing/pipelines. This will be easier via a parameter vs. renaming the files.